AnnoyancesFilter/sections/subscriptions.txt

```diff
@@ -37,6 +37,7 @@ turkuazgazetesi.net,yenialanya.com,mansetaydin.com,seskocaeli.com,kocaelibarisga
 !
 peopletalk.ru##body .zen-block
 ukr.media##.push_btn
+folhabv.com.br##iframe[src$="/forms"]
```

Comment:
Better block by URL in addition to element hiding, ~7 MB
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/8361299/167195790-cf67922e-5d76-4841-b5e5-868f46984914.png)

</details>